motoman_driver/MotoPlus/Controller.c

```diff
@@ -198,13 +136,6 @@ BOOL Ros_Controller_Init(Controller* controller)
 	
 	// wait for controller to be ready for reading parameter
 	Ros_Controller_WaitInitReady(controller);
-	
-	bInitOk = Ros_Controller_CheckSetup();
-	if (!bInitOk)
-		return FALSE; //Don't allow initialization to continue
-
-	// Wait for alarms to clear, in case Ros_Controller_CheckSetup raised an alarm
-	Ros_Controller_WaitInitReady(controller);

```

Comment:
Is this not needed at all any more?

What about other causes of alarms, separate from the `CheckSetup(..)`?

Reply:
This has been embedded in the parameter extraction code.